saos-batch/src/test/resources/import/constitutionalTribunal/judgments/version2/ctJudgments_v2_1.json

```diff
@@ -0,0 +1,186 @@
rokiego uczestnictwa w przygotowywaniu i opiniowaniu prawa, które ma obowiązywać we wszystkich państwach członkowskich (pojawiała się nawet koncepcja Kongresu Narodów – dodatkowego organu, będącego reprezentacją parlamentów krajowych na szczeblu Unii). Ma to tym istotniejsze znaczenie, że powszechnie dostrzegane jest zjawisko „podwójnego braku demokracji” (zwane też „deficytem demokracji”), polegające z jednej strony na utracie kompetencji prawotwórczych przez parlamenty narodowe, z drugiej zaś na nierównoważeniu tej utraty kompetencjami Parlamentu Europejskiego. Rodzić to może poczucie bezwartościowości głosów wyborczych obywateli państw Unii (zarówno w krajowych wyborach parlamentarnych, jak i w wyborach do Parlamentu Europejskiego), którzy w praktyce tracą wpływ na kierunki rozwoju Unii Europejskiej oraz na kształt rozwiązań prawnych, bowiem prawo unijne w większości powstaje w drodze działań władz wykonawczych, zmiennych konstelacji wspólnych interesów oraz niełatwych do zrozumienia kompromisów. Temu właśnie odczuciu przeciwdziałać ma dążenie do jednoznacznego określenia i zwiększenia udziału parlamentów krajowych w procesie stanowienia wspólnego prawa europejskiego.\n\n5. Organizacja udziału parlamentów narodowych w procesie stanowienia prawa unijnego jest zróżnicowana i należy do konstytucyjnej praktyki każdego z państw. Stopień skrępowania rządów decyzjami (opiniami) parlamentów jest różny, a ich kompetencje mogą obejmować zakres od wymiany informacji i ewentualnych nieformalnych uzgodnień (np. Irlandia, Hiszpania, Włochy) aż po narzucanie wiążącego stanowiska (np. Dania, Austria). Ponadto różny może być zakres przedmiotowy kompetencji, bowiem wszystkie komisje parlamentarne są właściwe w sprawach należących do I filaru (wspólny rynek, unia celna, wspólne reguły konkurencji, liczne wspólne polityki), natomiast w sprawach wspólnej polityki zagranicznej i bezpieczeństwa (II filar – Common Foreign and Security Policy; CFSP) oraz współpracy policyjnej i sądowej w sprawach karnych (III filar – dawniej obejmujący całość zagadnień sprawiedliwości i spraw wewnętrznych – Justice and Home Affairs, z których część została włączona do I filaru) właściwe są komisje jedynie w połowie państw członkowskich.\nPodkreślenia wymaga też, że nawet w państwach, których parlamenty udzielają rządom mandatów negocjacyjnych, władza wykonawcza mimo wszystko dysponuje pewnym zakresem swobody, niezbędnym do elastycznego reagowania na wyniki prowadzonych negocjacji. W doktrynie wyróżnia się trzy grupy parlamentów:\n\tpodejmujące decyzje polityczne (dysponujące prawem veta, a także możliwością wprowadzania zmian i zawierania kompromisów w procesie podejmowania decyzji),\n\twpływające na decyzje polityczne (mające prawo zmian projektów, ale bez prawa ich odrzucenia),\n\tsłabe (pozbawione możliwości zmiany czy odrzucenia projektów przedłożonych przez władzę wykonawczą).\nNa tym tle za celowe uznać można zwięzłe przedstawienie zasadniczych koncepcji wpływu parlamentów narod
```

Comment:
And have you used at least a few real judgments from files from Michał or have you created each of them by hand?